evansmusomi/burger-builder
# Forwards Unauthenticated Users

*What this does*
- Forwards unauthenticated users to auth screen
- Redirects unauthenticated users who'd begun building burgers to the checkout page after sign in

*How to verify*
- Signed in users should be able to build and order a burger as before
- Unauthenticated users should be able to "sign up to order"
- Users signing up to order should be redirected the checkout page with their burger preview
